src/main/docker/server.xslt

```diff
@@ -57,9 +57,7 @@
     <!-- Override the specified access logging element -->
     <xsl:template match="/Server/Service[@name='Catalina']/Engine[@name='Catalina']/Host[@name='localhost']/Valve[@className='org.apache.catalina.valves.AccessLogValve']">
         <xsl:copy>
-            <xsl:attribute name="className">
-                <xsl:value-of select="@className" />
-            </xsl:attribute>
+            <xsl:attribute name="className">ch.qos.logback.access.tomcat.LogbackValve</xsl:attribute>
             <xsl:attribute name="directory">/dev</xsl:attribute>
             <xsl:attribute name="prefix">stdout</xsl:attribute>
             <xsl:attribute name="rotatable">false</xsl:attribute>
```

Comment:
Are all these other parameters (i.e. `directory`, `prefix`, `rotatable`, etc.) relevant with this new valve?

Reply:
This comment alone pending. Working on it. Other comments are fixed.